translator.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+"""Translates any text document into an .fb file
+
+First CLI argument should be a text file to be converted.
+Print to `.fb` file extension with ` > output_file.fb`
+
+Example:
+
+./translator.py original_text_file.txt > new_fb_file.fb
+
+Check translation with ./code_runner.py new_fb_file.fb"""
+
+import sys
+
+
+class Translator():
+    """Prepares common FizzBuzzLang functions and retrieves file text content
+
+    ---
+    Attributes:
+        look_up: dict -- self-populating dictionary of finary refs
+        INPUT: str -- text content of file passed in CLI
+        STORE_CHAR: str -- allows storing of character at data-space location
+        MOVE_FORWARD: str -- move pointer forward one space in data-space
+        STORE_POS_FIZZ: str -- stores current data-space location in FIZZ loc.
+        END_OF_CODE: str -- prints data
+        TERMINATE_COMMAND: str -- termination command to end fbi
+        FBL_TEXT_ARRAY: list -- INPUT written in finary
+    """
+
+    def __init__(self) -> None:
+        self.look_up = {}
+        self.INPUT = self._retrieve_file()
+        self.STORE_CHAR = "BUZZ FIZZBUZZ FIZZBUZZ "
+        self.MOVE_FORWARD = "FIZZ FIZZ FIZZ"
+        self.STORE_POS_FIZZ = "FIZZ FIZZBUZZ FIZZ"
+        self.END_OF_CODE = """BUZZ FIZZBUZZ FIZZBUZZ BUZZ FIZZ BUZZ FIZZ
+FIZZ FIZZBUZZ FIZZBUZZ FIZZ
+FIZZBUZZ FIZZ FIZZBUZZBUZZ
+BUZZ BUZZ
+FIZZ FIZZ FIZZ
+FIZZBUZZ BUZZ FIZZ FIZZBUZZBUZZ"""
+        self.TERMINATE_COMMAND = "FIZZBUZZ FIZZBUZZ"
+        self.FBL_TEXT_ARRAY = self._convert_to_fbl()
+
+    def _retrieve_file(self):
+        """Retrieves text from file
+
+        Accepts CLI argument
```

Comment:
I'd be against having an internal method deal with external parameters. I would suggest you instead get the parameter in the main function, and pass it here explicitly

Reply:
That makes sense, I will refactor this to sit inside the main function. Perhaps that would also resolve your comment below regarding `exit` 
https://github.com/lechien73/FizzBuzzLang/pull/3/files/0eeeced3a5637dfc04bbabfb2ed170ed16b407a3#r857816764